Assets/Mirror/Runtime/NetworkSceneManager.cs

```diff
@@ -50,7 +50,7 @@ public class NetworkSceneManager : MonoBehaviour, INetworkSceneManager
         /// <para>This is used to make sure that all scene changes are initialized by Mirror.</para>
         /// <para>Loading a scene manually wont set networkSceneName, so Mirror would still load it again on start.</para>
         /// </remarks>
-        public string NetworkSceneName { get; protected set; } = "";
+        public string NetworkSceneName => SceneManager.GetActiveScene().name;
```

Comment:
please .path instead of .name

The reason is that we normally have a lot of scenes with the same name in different folders

Reply:
I will swap this and rerun tests. Pretty sure there are places where its assumed to be name.

Reply:
> What happens if I only have one scene for everything (like in basic example)?

Since we send the current scene to the client upon connect it will always send the client through the scene change process. This is mostly to ensure that both client and server are infact in the same scene. Since this is a Normal scene change this was added:

```
case SceneOperation.Normal:
//Scene is already active.
if (NetworkSceneName.Equals(sceneName))
{
    FinishLoadScene(sceneName, sceneOperation);
}
```

This skips the actual changing of the scene and just calls the post change events.